workstation/299_setup-completed.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+
+# Add a dot file to the home directory indicating that the setup has been completed successfully.
+# The host-side of the connection will look for this file when polling for completion to indicate to
```

Comment:
For my understanding, where is this defined upstream? Is it some setting in GC workstation config?

Reply:
The `sentry workstations create` tool knows to look for this file specifically. That PR is incoming in the `sentry` repo.